crypto/keyring/keyring.go

```diff
@@ -327,6 +331,15 @@ func (ks keystore) ImportPubKey(uid string, armor string, keyType KeyType) error
 	return nil
 }
 
+// MigrateInfo implements Importer.MigrateInfo.
+func (ks keystore) MigrateInfo(oldInfo Info) error {
```

Comment:
I'm passing the old `Import{}Key` methods, and replacing both of them with `MigrateInfo`.

### Before

The old migration logic was: we read data from the old Info, decode it (depending on the type), extract pubkey, and create and write a new Info in the new keyring.

In the process, we completely lose the Info: we were migrating old TypeMulti and old TypeLedger into new TypeOffline. https://github.com/cosmos/cosmos-sdk/pull/8639#discussion_r582962243

### After

We don't decode Info anymore (to extract pubkey etc). Since the infos didn't change, we just copy/paste the old Info into the keyring. It's imo much simpler.